Cargo.toml

```
@@ -21,12 +21,14 @@ include = [
 edition = "2021"
 
 [dependencies]
-embedded-hal = "0.2"
+embedded-hal = "1"
 rtcc = "0.3"
 
 [dev-dependencies]
-linux-embedded-hal = "0.3"
-embedded-hal-mock = "0.8"
+linux-embedded-hal = "0.4"
+embedded-hal-mock = { version = "0.11.1", default-features = false, features = [
+    "eh1",
```

```suggestion
    "eh1"
```
Maybe cargo does not like the comma there?